tests/imap/test_delete_handling.py

```diff
@@ -301,9 +301,9 @@ def test_renamed_label_refresh(
 
     new_flags = {
         msg_uid: {
-            "FLAGS": ("\\Seen",),
-            "X-GM-LABELS": ("new label",),
-            "MODSEQ": ("23",),
+            b"FLAGS": (b"\\Seen",),
+            b"X-GM-LABELS": (b"new label",),
+            b"MODSEQ": (23,),
```

Comment:
This tests Gmail, in Gmail modseqs are always integers, so simulate real life better.

Note that generic IMAP can return this as something, but we always normalize it somewhere in the code.